scripts/breaking_changes_checker/detect_breaking_changes.py

```diff
@@ -348,9 +349,13 @@ def main(
         client = PyPIClient()
 
         try:
-            version = str(client.get_relevant_versions(package_name)[1])
+            if latest_pypi_version:
+                versions = client.get_ordered_versions(package_name)
+                version = str(versions[-1]) if len(versions) > 0 else None
+            if not version:
+                version = str(client.get_relevant_versions(package_name)[1])
```

Comment:
I'm having a little trouble understanding this -- if get_ordered_versions returns [], wouldn't get_relevant_versions return the same? should it be something like...

```suggestion
            if latest_pypi_version:
                versions = client.get_ordered_versions(package_name)
                version = str(versions[-1])
            else:
                version = str(client.get_relevant_versions(package_name)[1])
```

Reply:
You're right, fixed it :) thanks!